pages/attacks/Embedding_Null_Code.md

```diff
@@ -3,7 +3,7 @@
 title: Embedding Null Code
```

Comment:
```suggestion
title: 'Embedding Null Code'
```

Reply:
Good to know it requires being wrapped - I assumed it might not matter since it's only one string of text.